src/server/tests.rs

```diff
@@ -2948,16 +2953,16 @@ csv.from(file: "my.csv")
                 lsp::SemanticToken {
                     delta_line: 1,
                     delta_start: 9,
-                    length: 12,
+                    length: 0,
                     token_type: SEMANTIC_TOKEN_STRING,
                     token_modifiers_bitset: 0,
                 },
                 // import
                 lsp::SemanticToken {
-                    delta_line: 2,
-                    delta_start: 1,
-                    length: 6,
-                    token_type: SEMANTIC_TOKEN_KEYWORD,
+                    delta_line: 1,
+                    delta_start: 9,
+                    length: 12,
+                    token_type: SEMANTIC_TOKEN_STRING,
```

Comment:
Just wanted to call out this change explicitly. This is what gets the test to pass, but I'm not entirely sure what necessitated this change, and it looked suspicious. To whoever reviews this: please confirm this is correct :pray:

Reply:
Thanks for pointing this out. I pulled this branch to my local machine and ran a test. It works fine!